packages/rum-core/src/domain/rumEventsCollection/view/trackViewMetrics.spec.ts

```diff
@@ -1,14 +1,15 @@
-import { LifeCycleEventType, RumEvent } from '@datadog/browser-rum-core'
 import { Context, RelativeTime, Duration } from '@datadog/browser-core'
+import { LifeCycleEventType, RumEvent } from '@datadog/browser-rum-core'
 import { TestSetupBuilder, setup } from '../../../../test/specHelper'
-import { RumEventType } from '../../../rawRumEvent.types'
 import { RumPerformanceNavigationTiming } from '../../../browser/performanceCollection'
+import { RumEventType } from '../../../rawRumEvent.types'
 import {
   PAGE_ACTIVITY_END_DELAY,
   PAGE_ACTIVITY_MAX_DURATION,
   PAGE_ACTIVITY_VALIDATION_DELAY,
 } from '../../trackPageActivities'
-import { ViewEvent, trackViews, THROTTLE_VIEW_UPDATE_PERIOD } from './trackViews'
+import { THROTTLE_VIEW_UPDATE_PERIOD } from './trackViews'
+import { setupViewTest, ViewTest } from './trackViews.spec'
```

Comment:
`.spec` files should not be imported, because their tests will be executed twice

Reply:
oh, I was not aware of that 🤔 
nice strategy to bump the number of tests 😆 

maybe we could add a lint check on that

Reply:
It is caused by our webpack configuration when running karma: https://github.com/DataDog/browser-sdk/blob/2f661f2e66b49224b627921cd6887096ce05c3b0/test/unit/karma.base.conf.js#L49-L53 (this was the default with the previous version of karma-webpack). We need this because some tests are not properly cleaned-up. We may investigate how to improve the situation one day :)

Reply:
what do you think of the lint rule in the mean time?

Reply:
It wouldn't hurt!